src/components/ExpenseList.js

```diff
@@ -4,23 +4,35 @@ import Expense from './Expense.js';
 
 export default function ExpenseList() {
   const [expenses, setExpenses] = useState([
-    { title: 'Chinese food', amount: '20.00€' },
-    { title: 'Taxi', amount: '15.00€' },
+    { title: 'Chinese food', amount: 20.0 },
+    { title: 'Taxi', amount: 15.74 },
   ]);
-  const [inputValue, setInputValue] = useState('');
+  const [inputValue, setInputValue] = useState({ title: '', amount: '' });
   const handleSubmit = e => {
     e.preventDefault();
     setExpenses(prevExpenses => [
       ...prevExpenses,
       {
-        title: inputValue,
-        amount: '32.00€',
+        title: inputValue.title,
+        amount: inputValue.amount,
       },
     ]);
-    setInputValue('');
+    setInputValue({ title: '', amount: '' });
+    console.log(inputValue.amount.type);
+    console.log(totalExpenses.type);
   };
+  const totalExpenses = Number(
+    expenses
+      .map(expense => expense.amount)
+      .reduce((prev, curr) => +prev + +curr, 0)
```

Comment:
Not sure if you need the double ++ in line 27

Reply:
Necessary, otherwise the values are read as strings. Tried it without.